tests/esp32/wait_for2.toit

```diff
@@ -17,10 +17,12 @@ main:
   pin_in := gpio.Pin PIN_IN --input
   pin_out := gpio.Pin PIN_OUT --output
 
-  ITERATIONS.repeat:
+  ITERATIONS.repeat: | iteration |
+    if iteration % 1000 == 0: print "Iteration: $iteration"
     before := pin_in.get
     with_timeout --ms=2_000:
       pin_in.wait_for 1
+    if pin_in.get != 1: throw "Expected pin to be 1 - $iteration $before"
```

Comment:
The test here generally does not expect or throw. This is different. Should this use 'expect'?

Reply:
Switched to `expect`.